router/src/grpc_server.rs

```diff
@@ -545,10 +545,11 @@ fn convert_params(
             if let Some(d) = p.decoding {
                 if d.repetition_penalty != 0.0 {
                     gp.repetition_penalty = d.repetition_penalty;
-                    gp.length_penalty =
-                        d.length_penalty.map(|lp| (lp.start_index, lp.decay_factor));
                 }
+                gp.length_penalty =
+                    d.length_penalty.map(|lp| (lp.start_index, lp.decay_factor));
             }
+
```

Comment:
nit: no need for this extra line, may as well remain consistent with the surrounding code